shared/redux/rtk-apis/exams/exams.api.ts

```diff
@@ -0,0 +1,20 @@
+import { TApiResponse } from "@/shared/typedefs";
+
+import projectApi from "../api.config";
+import { TCreateExamDto, TExam } from "./exams.types";
+
+const examsApi = projectApi.injectEndpoints({
+  endpoints: (builder) => ({
+    createExam: builder.mutation<TExam, { id: number; newExam: TCreateExamDto }>({
+      query: ({ id, ...rest }) => ({
+        url: `classrooms/${id}/exams`,
+        method: "POST",
+        body: rest.newExam,
```

Comment:
You can just do this 


```suggestion
      query: ({ id, newExam }) => ({
        url: `classrooms/${id}/exams`,
        method: "POST",
        body: newExam,
```